docs/en/latest/plugins/openid-connect.md

```diff
@@ -61,6 +61,7 @@ description: OpenID Connect allows the client to obtain user information from th
 | session                              | object  | False    |                       |              | When bearer_only is set to false, openid-connect will use Authorization Code flow to authenticate on the IDP, so you need to set the session-related configuration. |
 | session.secret                       | string  | True     | Automatic generation  | 16 or more characters | The key used for session encrypt and HMAC operation. |
 | unauth_action                        | string  | False    | "auth"                |              | Specify the response type on unauthenticated requests. "auth" redirects to identity provider, "deny" results in a 401 response, "pass" will allow the request without authentication. |
+| proxy_opts                           | object  | False    |                       |              | Configure an HTTP proxy to be used with the openid-connect plugin. |
```

Comment:
Could you list all options?

Reply:
> Could you list all options?

done
add no-proxy refer to 
https://github.com/ledgetech/lua-resty-http#set_proxy_options

Reply:
I didn't see the options in `proxy_opts`

Reply:
The option you added in the doc `proxy_opts`, it should have some attributes as you described in the schema.
The attributes should be added to the doc too, or the users don't know how to use the `proxy_opts`
```
        proxy_opts = {
            description = "access openid server via a proxy server ",
            type = "object",
            properties = {
                http_proxy = {
                    type = "string",
                    description = "Http proxy: http://proxy-server:80",
                },
                https_proxy = {
                    type = "string",
                    description = "Https proxy: https://proxy-server:80",
                },
                http_proxy_authorization = {
                    type = "string",
                    description = "Basic [base64 username:password].",
                },
                https_proxy_authorization = {
                    type = "string",
                    description = "Basic [base64 username:password].",
                },
                no_proxy = {
                    type = "string",
                    description = "A comma separated list of hosts that should not be proxied.",
                }
            },
```

Reply:
hi @monkeyDluffy6017 
I just submitted the modification, please help to review it again when it is convenient

Reply:
hi @Sn0rt 
by the way
I noticed that you modified the desc in openid-connect schema
 by using `https://proxy-server` instead of  `http://proxy-server`  for  `https_proxy` .
but it will leads to err `protocol https not supported for proxy connections` when i config the `proxy_opts` for our app before. 
And I fix it by set `https_proxy` = `http://proxy-server`  instead of `https://proxy-server` 
We may need to confirm this disagreement